django_mongodb_engine/compiler.py

```diff
@@ -388,9 +391,9 @@ def insert(self, docs, return_id=False):
         options = self.connection.operation_flags.get('save', {})
 
         if return_id:
-            return collection.save(doc, **options)
+            return collection.insert_one(doc, **options)
```

Comment:
@kondaiahv : Looks like the function insert can pass single or multiple docs.
So we need to support both insert_one and insert_many()?
How the old code is working also we need to check

Reply:
Actual Django-mongo save(update/insert) only one doc at a time based on primary key(_id). If we want to update multiple documents, we need to change this logic & our code(if anywhere handling return ids).